ocean_lib/models/test/test_erc20_token.py

```diff
@@ -14,35 +14,34 @@
 from ocean_lib.web3_internal.constants import MAX_UINT256, ZERO_ADDRESS
 from ocean_lib.web3_internal.currency import to_wei
 from ocean_lib.web3_internal.utils import split_signature
-from tests.resources.helper_functions import deploy_erc721_erc20, get_address_of_type
+from tests.resources.helper_functions import get_address_of_type
 
 
 @pytest.mark.unit
-def test_properties(web3, config, publisher_wallet):
+def test_properties(web3, config, publisher_wallet, erc20_token):
     """Tests the events' properties."""
-    _, erc20 = deploy_erc721_erc20(
-        web3=web3,
-        config=config,
-        erc721_publisher=publisher_wallet,
-        erc20_minter=publisher_wallet,
-        cap=to_wei("1"),
-    )
-
-    assert erc20.event_NewPool.abi["name"] == ERC20Token.EVENT_NEW_POOL
-    assert erc20.event_NewFixedRate.abi["name"] == ERC20Token.EVENT_NEW_FIXED_RATE
-    assert erc20.event_MinterProposed.abi["name"] == ERC20Token.EVENT_MINTER_PROPOSED
-    assert erc20.event_OrderStarted.abi["name"] == ERC20Token.EVENT_ORDER_STARTED
-    assert erc20.event_MinterApproved.abi["name"] == ERC20Token.EVENT_MINTER_APPROVED
-    assert erc20.event_OrderReused.abi["name"] == ERC20Token.EVENT_ORDER_REUSED
-    assert erc20.event_OrderExecuted.abi["name"] == ERC20Token.EVENT_ORDER_EXECUTED
+    assert erc20_token.event_NewPool.abi["name"] == ERC20Token.EVENT_NEW_POOL
+    assert erc20_token.event_NewFixedRate.abi["name"] == ERC20Token.EVENT_NEW_FIXED_RATE
     assert (
-        erc20.event_PublishMarketFeeChanged.abi["name"]
+        erc20_token.event_MinterProposed.abi["name"] == ERC20Token.EVENT_MINTER_PROPOSED
+    )
+    assert erc20_token.event_OrderStarted.abi["name"] == ERC20Token.EVENT_ORDER_STARTED
+    assert (
+        erc20_token.event_MinterApproved.abi["name"] == ERC20Token.EVENT_MINTER_APPROVED
+    )
+    assert erc20_token.event_OrderReused.abi["name"] == ERC20Token.EVENT_ORDER_REUSED
+    assert (
+        erc20_token.event_OrderExecuted.abi["name"] == ERC20Token.EVENT_ORDER_EXECUTED
+    )
+    assert (
+        erc20_token.event_PublishMarketFeeChanged.abi["name"]
         == ERC20Token.EVENT_PUBLISH_MARKET_FEE_CHANGED
     )
```

Comment:
Did black do these changes? Yuck, it's so much harder to read

Reply:
Yep, it's due to black formatting.